spec/rails_helper.rb

```diff
@@ -488,9 +488,12 @@ def synchronize(seconds = nil, errors: nil)
       end
     end
 
+    # Sets sequence's value to be greater than the max value that an INT column can hold. This is done to prevent
+    # type mistmatches for foreign keys that references a column of type BIGINT. We set the value to 10_000_000_000
+    # instead of 2**32 so that the values are easier to read.
```

Comment:
Technically the maximum value is `2**31-1`, but I prefered `2**32` because it is shorter to write.

```suggestion
    # instead of 2**31-1 so that the values are easier to read.
```